Program.cs

```diff
@@ -3,18 +3,67 @@
 // Copyright (c) Metamation India.                                                
 // ---------------------------------------------------------------------------
 // Program.cs                                                                     
-// 
-// 
-// 
+// Program to help solve a New-York Times style Spelling Bee.
+// A word list is given as a text file, and that the daily choice of 7 letters is provided as an array of 7 chars:
+// char[] letters = { 'U', 'X', 'A', 'L', 'T', 'N', 'E' }
+// Valid word conditions - Each word is at least 4 letters long, Each word uses only the 7 seed letters,
+// Each word MUST use the first letter (in this case U),Any letter can be used more than once.
+// Score conditions - 4 letter words score 1 point, Longer words score their no. of letters as points,
+// Any word that uses all 7 seed letters (called pangrams) gets an additional 7 bonus points.
+// Output is a list of words sorted in descending order of score.
+// If two words have the same score, they are displayed in alphabetical order of the words.
+// Pangrams are displayed in green color
 // ---------------------------------------------------------------------------------------
 using static System.Console;
+
 namespace Training {
    #region class Program ------------------------------------------------------------------------
+   /// <summary>Spelling Bee</summary>
    internal class Program {
       #region Method ----------------------------------------------
-      static void Main (string[] args) {
-         Console.WriteLine ("Hello, World!");
+      /// <summary>This method reads the text file, displays valid words in the arrangement of (score. word) 
+      /// and displays pangrams in green color</summary>
+      static void Main () {
+         string[] words = File.ReadAllLines ("C:words.txt.crdownload");

```

Comment:
This will not run for me until I have a file with a similar name. Have a `.txt` file instead of `.crdownload` files. Set your file path to `C:/etc/words.txt` until we start using embedded resources.